.github/workflows/Bundle.yml

```
@@ -0,0 +1,80 @@
+name: Bundle PicoStation
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)